torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -603,7 +605,7 @@ def prepare_for_partitioner(mod, num_primals, num_fw_outputs):
 
                 new_hop_graphs[identifier].partitioning_done = True
 
-    if not new_hop_graphs:
+    if not new_hop_graphs or not found_backward_node:
```

Comment:
is this change related to the caching? (what is it for?)

Reply:
Not related to caching. It just checks the case where the backward graph is empty. Got exposed by the added trsrcase